TWLight/users/helpers/editor_data.py

```diff
@@ -160,14 +264,16 @@ def editor_recent_edits(
             # This means that eligibility always lasts at least 30 days.
             or (wp_enough_recent_edits and editcount_update_delta.days > 30)
         ):
-            wp_editcount_recent = global_userinfo_editcount - wp_editcount_prev
-            wp_editcount_prev = global_userinfo_editcount
+            # recent edits = global userinfo editcount - stored editcount.
```

Comment:
Nitpick: maybe this comment is redundant (at least to me).

Reply:
Fair. The comment preceded the code, and was mostly there to help me sort the bug out.
```suggestion
```